### PR TITLE
Cypress: new TapRegex test is ready + pageObjects

### PR DESCRIPTION
**Pull request by AdamKol-up9**

### CODE REVIEW

File: acceptanceTests/cypress/integration/page_objects/StatusBar.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                // organizing namespaces array
+                const namespacesObjectsArray = Object.values(namespaces)
+                let namespacesArray = []
+                namespacesObjectsArray.forEach(line => {
+                    line.getAttribute ? namespacesArray.push(line.innerHTML) : null
+                })
+
+                // organizing pods array
+                const podObjectsArray = Object.values(pods)
+                let podsArray = []
```
Comment (RoyUP9) [inline acceptanceTests/cypress/integration/page_objects/StatusBar.js:34]: array to const

File: acceptanceTests/cypress/integration/page_objects/StatusBar.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+const columns = {"podName" : 1, "namespace" : 2, "tapping" : 3}
+const greenStatusImageSrc = "/static/media/success.662997eb.svg"
+
+class StatusBarFunctions {
```
Comment (RoyUP9) [inline acceptanceTests/cypress/integration/page_objects/StatusBar.js:4]: class not needed

File: acceptanceTests/cypress/integration/page_objects/StatusBar.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        })
+    }
+
+    findLineAndCheck(expectedValues) {
+        cy.get('.expandedStatusBar > :nth-child(2) > > :nth-child(2) > > :nth-child(1)').then(pods => {
+            cy.get('.expandedStatusBar > :nth-child(2) > > :nth-child(2) > > :nth-child(2)').then(namespaces => {
+
+                // organizing namespaces array
+                const namespacesObjectsArray = Object.values(namespaces)
+                let namespacesArray = []
```
Comment (RoyUP9) [inline acceptanceTests/cypress/integration/page_objects/StatusBar.js:27]: array to const

File: acceptanceTests/cypress/integration/page_objects/StatusBar.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
+const columns = {"podName" : 1, "namespace" : 2, "tapping" : 3}
```
Comment (RoyUP9) [inline acceptanceTests/cypress/integration/page_objects/StatusBar.js:1]: decide if " or '

File: acceptanceTests/cypress/integration/page_objects/StatusBar.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+            expect(podName).to.equal(expectedValues.podName)
+
+            cy.get(this.getDomPathInStatusBar(line, columns.namespace)).invoke('text').then(namespaceValue => {
+                expect(namespaceValue).to.equal(expectedValues.namespace)
+                cy.get(this.getDomPathInStatusBar(line, columns.tapping)).children().should('have.attr', 'src', greenStatusImageSrc)
+            })
+        })
+    }
+
+    findLineAndCheck(expectedValues) {
```
Comment (RoyUP9) [inline acceptanceTests/cypress/integration/page_objects/StatusBar.js:21]: refactor to func

File: acceptanceTests/cypress/integration/page_objects/StatusBar.js [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+            })
+        })
+    }
+
+    throwError(pod, namespace) {
+        throw new Error(`The pod named ${pod} doesn't match any namespace named ${namespace}`)
+    }
+
+    getExpectedDetailsDict(podName, namespace) {
+        return {"podName" : podName, 'namespace' : namespace}
```
Comment (RoyUP9) [inline acceptanceTests/cypress/integration/page_objects/StatusBar.js:55]: remove " and '

File: acceptanceTests/cypress/integration/page_objects/StatusBar.js [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+        })
+    })
+}
+
+export function findLineAndCheck(expectedValues) {
+    cy.get('.expandedStatusBar > :nth-child(2) > > :nth-child(2) > > :nth-child(1)').then(pods => {
+        cy.get('.expandedStatusBar > :nth-child(2) > > :nth-child(2) > > :nth-child(2)').then(namespaces => {
+
+            // organizing namespaces array
+            const podObjectsArray = Object.values(pods)
```
Comment (lirazyehezkel) [inline acceptanceTests/cypress/integration/page_objects/StatusBar.js:25]: (pods ?? {})

File: acceptanceTests/cypress/integration/page_objects/StatusBar.js [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+            expect(namespaceValue).to.equal(expectedValues.namespace)
+            cy.get(getDomPathInStatusBar(line, columns.tapping)).children().should('have.attr', 'src', greenStatusImageSrc)
+        })
+    })
+}
+
+export function findLineAndCheck(expectedValues) {
+    cy.get('.expandedStatusBar > :nth-child(2) > > :nth-child(2) > > :nth-child(1)').then(pods => {
+        cy.get('.expandedStatusBar > :nth-child(2) > > :nth-child(2) > > :nth-child(2)').then(namespaces => {
+
```
Comment (RoyUP9) [inline acceptanceTests/cypress/integration/page_objects/StatusBar.js:23]: remove space

File: acceptanceTests/cypress/integration/page_objects/StatusBar.js [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+            // organizing namespaces array
+            const podObjectsArray = Object.values(pods)
+            const namespacesObjectsArray = Object.values(namespaces)
+            let rightIndex = -1;
+            namespacesObjectsArray.forEach((namespaceObj, index) => {
+                rightIndex = (namespaceObj.getAttribute && namespaceObj.innerHTML === expectedValues.namespace && (podObjectsArray[index].innerHTML.substring(0, podObjectsArray[index].innerHTML.indexOf('-'))) === expectedValues.podName) ? index + 1 : rightIndex
```
Comment (RoyUP9) [inline acceptanceTests/cypress/integration/page_objects/StatusBar.js:29]: function

**Review by RoyUP9 — APPROVED** [review]

**Pull request merged**